contents/blog/0.1-release/index.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Introducing OpenLineage 0.1
```

Comment:
Officially, it's `0.1.0` see https://github.com/OpenLineage/OpenLineage/releases/tag/0.1.0